.goreleaser.yml

```diff
@@ -60,6 +60,14 @@ brews:
     test: |
       help_text = shell_output("#{bin}/fastly --help")
       assert_includes help_text, "Usage:"
+scoop:
+  bucket:
+    owner: fastly
+    name: scoop-cli
+  homepage: https://github.com/fastly/cli
+  skip_upload: auto
+  description: Fastly CLI
+  license: Apache 2.0
```

Comment:
https://github.com/fastly/scoop-cli/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R29 says "MIT"

Reply:
nvm this License is for fastly/cli itself, obviously.